lesson08/assignment/lesson08/src/components/accordion.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+<body>
+
+    <div class="container">
+        <h2 class="m-3">Frequently asked questions</h2>
+        <!--Use v-for to populate accordion-->
+        <div v-for="item in items" class="accordion" id="accordion">
+            <div class="card"> 
+              <div class="card-header" id="headingOne">
+                <h2>
+                  <button class="btn btn-link" type="button" v-on:click="show = !show" data-target="#collapseOne">
+                    {{ item.faq }}
+                  </button>
+                </h2>
+              </div>
+
+              <div id="collapseOne" v-if="show" class="collapse show" data-parent="#accordion">
```

Comment:
This is the one that isn't working as expected, right? Each of the answers is hiding/showing based on the value of the same variable, `show`. I'd recommend making the variable name or the div identifier unique based on the item's index in the array. Here's one approach: https://stackoverflow.com/questions/45082800/how-to-properly-pass-index-in-bootstrap-data-target-v-for